tests/e2e/performance/test_small_file_workload.py

```diff
@@ -323,9 +317,124 @@ class TestSmallFileWorkload(PASTest):
     """
 
     def setup(self):
-        super(TestSmallFileWorkload, self).setup()
+        """
+        Setting up test parameters
+        """
+        log.info("Starting the test setup")
         self.benchmark_name = "SmallFiles"
         self.client_pod_name = "smallfile-client"
+        if config.PERF.get("deploy_internal_es"):
+            self.es = ElasticSearch()
+        else:
+            if config.PERF.get("internal_es_server") == "":
+                self.es = None
+                return
+            else:
+                self.es = {
+                    "server": config.PERF.get("internal_es_server"),
+                    "port": config.PERF.get("internal_es_port"),
+                    "url": f"http://{config.PERF.get('internal_es_server')}:{config.PERF.get('internal_es_port')}",
+                }
+                # verify that the connection to the elasticsearch server is OK
+                if not super(TestSmallFileWorkload, self).es_connect():
+                    self.es = None
+                    return
+
+        super(TestSmallFileWorkload, self).setup()
+        # deploy the benchmark-operator (ripsaw)
+        self.ripsaw = RipSaw()
+        self.ripsaw_deploy(self.ripsaw)
+
+    def setting_storage_usage(self, file_size, files, threads, samples):
```

Comment:
Doc string required for the param in function setting_storage_usage()

Reply:
done.

Reply:
done.